Fix: Snapshot location might differ from that of BQ slots (#3448) (#3451)
DataBiosphere/azul: UPGRADING.rst

reverted. This is all fairly informal and loosely defined. Hopefully we won't
have too many entries in this file.

> #3448 Make snapshot location configurable

Reviewer: 2 new lines before a heading.

Reviewer: Does the heading have to match the commit title? I don't see anything within the contributing.rst..... 🤔

have too many entries in this file.

#3448 Make BQ slot location configurable
========================================

A new configuration variable has been added, ``AZUL_TDR_SOURCE_LOCATION``.

> Set the variable to the storage location of the snapshots you are indexing.

Reviewer 2: ```suggestion
Set the variable to the storage location of the snapshots the deployment is configured to index.
```

have too many entries in this file.

#3448 Make BQ slot location configurable
========================================

> A new configuration variable has been added, ``AZUL_TDR_SLOT_LOCATION``.

Reviewer 2: ```suggestion
A new configuration variable has been added, ``AZUL_TDR_SOURCE_LOCATION``.
```
